src/validator/has-length-greater-than.ts

```diff
@@ -0,0 +1,37 @@
+import {STRule} from '../rule/rule';
+import {STRuleFn} from '../rule/fn';
+import {STRuleModifiers} from '../rule/modifiers';
+import {STRuleNode} from '../rule/node';
+import {STRuleNodeType} from '../rule/node-type';
+import {greaterThanFn} from './is-greater-than';
+
+export type STOpHasLengthGreaterThan<CallerType> = (a: number) => CallerType;
+
+export const hasLengthGreaterThan = (curr: any, target: number): boolean => {
+	if (curr.length <= -1) {
```

Comment:
- [ ] Similar to above, simplify statement to `if (curr.length < 0) {`
- [ ] Move length check below curr.length type check.